UKHomeOffice/sgmr-service
# Add maintenance mode

## Description
Add maintenance mode functionality to sGMR.

## How to test

1. Start the app with env var `SGMR_MAINTENANCE=1` or `SGMR_MAINTENANCE=true`
2. You should see the maintenance page


## Developer Checklist

\* Required
